### PR TITLE
Update documentation and replace rinkeby with goerli

### LINKED ISSUE #512: Create new project problem.
Hi there, I have to tried to deploy the whole project to rinkeby with the document in repo, but after I open the website and try to create a new project, the final step is stucked.

* Here is my deploy logs:
```
$ npx hardhat run --network rinkeby scripts/deployRound.ts
 ·--------------------------------------|-------------·
 |  Contract Name                       ·  Size (Kb)  │
 ·······································|··············
 |  Address                             ·       0.08  │
 ·······································|··············
 │ ...                                  ·             │ 
 ·······································|··············
 |  VerifyTally                         ·       2.33  │
 ·--------------------------------------|-------------·
*******************
Deploying a new clr.fund instance of contracts!
*******************
deployer.address:  0xB42faBF7BCAE8bc5E368716B568a6f8Fdf3F84ec
maciFactory.address:  0x806EE0951A74E4A3DA61AEfb3600a9e54286A4B8
fundingRoundFactory.address:  0x82BA798f523fF464D8397493Ff97672416ED3A98
userRegistry.address:  0x7340E74cF375B9bDA775eCE372b1C8981164840a
recipientRegistry.address:  0x11Ef1898448376EEC4eE33Bbd83FA775271a4F08
serializedCoordinatorPrivKey:  macisk.236bf3b50b07b2444d5f946e8a88d1b8d7e4ac3f49f4d091152d8aaa4dcdc3c7
serializedCoordinatorPubKey:  macipk.bed1ddab4f3c710eee1fb561b617b16d75ba27967441b977a00ccfeef544c30a
```

* This is my `.env` in `contracts`:
```
# Recipient registry type for local deployment: simple, optimistic
RECIPIENT_REGISTRY_TYPE=simple

# Supported values: simple, brightid
USER_REGISTRY_TYPE=simple
# clr.fund (prod) or CLRFundTest (testing)
BRIGHTID_CONTEXT=clr.fund
# BrightId node addr that signs verifications. Node One uses this one
BRIGHTID_VERIFIER_ADDR=0xb1d71F62bEe34E9Fc349234C201090c33BCdF6DB

# JSON-RPC endpoint to the selected network
JSONRPC_HTTP_URL=https://eth-rinkeby.alchemyapi.io/v2/1YRsY....g1TU

# One of the two options
WALLET_MNEMONIC=attract ... hover
WALLET_PRIVATE_KEY=

# Token address for funding round .setToken
NATIVE_TOKEN_ADDRESS=0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea

# Required to use in the tally and finalize scripts
FACTORY_ADDRESS=0x806EE0951A74E4A3DA61AEfb3600a9e54286A4B8
ROUND_ADDRESS=0x82BA798f523fF464D8397493Ff97672416ED3A98
COORDINATOR_PK=macisk.236bf3b50b07b2444d5f946e8a88d1b8d7e4ac3f49f4d091152d8aaa4dcdc3c7
COORDINATOR_ETH_PK=macipk.bed1ddab4f3c710eee1fb561b617b16d75ba27967441b977a00ccfeef544c30a
```

* This is my `.env` in `vue-app`:
```
# Chain details where contract is deployed
VUE_APP_ETHEREUM_API_URL=https://eth-rinkeby.alchemyapi.io/v2/1YRsY5...g1TU
VUE_APP_ETHEREUM_API_CHAINID=4
VUE_APP_INFURA_ID=4521b5...1a2b
VUE_APP_IPFS_GATEWAY_URL=https://ipfs.io
VUE_APP_SUBGRAPH_URL=http://localhost:8000/subgraphs/name/daodesigner/clrfund

# Comma-separated list of URLs
VUE_APP_GUN_PEERS=http://localhost:8765/gun

VUE_APP_CLRFUND_FACTORY_ADDRESS=0x82BA798f523fF464D8397493Ff97672416ED3A98

# Supported values: simple, brightid
VUE_APP_USER_REGISTRY_TYPE=simple
# clr.fund (prod) or CLRFundTest (testing)
# Learn more about BrightID and context in /docs/brightid.md
VUE_APP_BRIGHTID_CONTEXT=clr.fund

# Supported values: simple, optimistic, kleros
VUE_APP_RECIPIENT_REGISTRY_TYPE=simple

VUE_APP_RECIPIENT_REGISTRY_POLICY=QmeygKjvrpidJeFHv6ywjUrj718nwtFQgCCPPR4r5nL87R

# Comma-separated list of IPFS hashes
VUE_APP_EXTRA_ROUNDS=

# Operator of clr.fund instance
VUE_APP_OPERATOR=

# Google Service Account credentials in JSON format
GOOGLE_APPLICATION_CREDENTIALS=
# Spreadsheet ID to send recipients data
VUE_APP_GOOGLE_SPREADSHEET_ID=
```

Because I don't really know what `VUE_APP_RECIPIENT_REGISTRY_POLICY`, `VUE_APP_SUBGRAPH_URL` and `VUE_APP_GUN_PEERS` to do, I think there may be some network setting error here?

### LINKED ISSUE #514: Remove/Replace references to rinkeby testnet as it will be deprecated in Q4 2022.
Here's the announcement of the testnet deprecation
https://blog.ethereum.org/2022/06/21/testnet-deprecation/

### PR DESCRIPTION
**Pull request by yuetloo**

### CODE REVIEW

File: contracts/.env.example [hunk 1]
```diff
@@ -9,7 +9,7 @@ BRIGHTID_CONTEXT=clr.fund
 BRIGHTID_VERIFIER_ADDR=0xb1d71F62bEe34E9Fc349234C201090c33BCdF6DB
```
Comment (yuetloo) [inline contracts/.env.example:9]: @adamstallard , do you know what the BRIGHTID_VERIFIER_ADDR is on georli?

File: contracts/contracts/snarkVerifiers/README.md [hunk 2]
```diff
@@ -4,4 +4,22 @@ Trusted setup:
...
 - 'medium' circuits: https://gateway.pinata.cloud/ipfs/QmRzp3vkFPNHPpXiu7iKpPqVnZB97wq7gyih2mp6pa5bmD
-- 'x32' circuits: https://gateway.pinata.cloud/ipfs/QmWSxPBNYDtsK23KwYdMtcDaJg3gWS3LBsqMnENrVG6nmc
\ No newline at end of file
+- 'x32' circuits: https://gateway.pinata.cloud/ipfs/QmWSxPBNYDtsK23KwYdMtcDaJg3gWS3LBsqMnENrVG6nmc
+- 'batch64' circuits: https://gateway.pinata.cloud/ipfs/QmbVzVWqNTjEv5S3Vvyq7NkLVkpqWuA9DGMRibZYJXKJqy
+
+
+Instead of downloading the above circuits from the ipfs, they can be generated using the MACI scripts.  For example, to build the x32 circuits on linux Ubuntu 22.04 + Node v16.13.2:
+
+```
```
Comment (yuetloo) [inline contracts/contracts/snarkVerifiers/README.md:14]: @auryn-macmillan Do you know where the `buildSnarks` script for batch64?

Comment (auryn-macmillan) [reply contracts/contracts/snarkVerifiers/README.md:14]: Yes, it is this script.

https://github.com/privacy-scaling-explorations/maci/blob/master/circuits/scripts/buildCustomSnarks.sh

This is the command to generate the batch64 circuits.
`./scripts/buildCustomSnarks.sh -s 32 -m 32 -v 3 -i 6 -b 64`

File: contracts/.env.example (hunk 1 above)
Comment (auryn-macmillan) [reply contracts/.env.example:9]: I spoke with Adam, he said that the address on Goerli will be the same address used for the other networks.

We will need to set up new BrightID contexts though.
We should essentially set up a separate clrfund context for each network that we (or others) might use. But this is out of scope for this PR and can be tracked elsewhere.

**Pull request merged**